EventFilter/test/gemTestBeam.py

```diff
@@ -48,10 +48,40 @@
 
 process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
 from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2021_realistic', '')
```

Comment:
this shouldnt be needed, it will just slow the init

Reply:
for magnetic field, https://github.com/cms-sw/cmssw/blob/master/MagneticField/Engine/python/uniformMagneticField_cfi.py can be used

Reply:
I've changed and pushed the fixed version.